osquery/tables/system/posix/yum_sources.cpp

```diff
@@ -81,15 +82,16 @@ QueryData genYumSrcs(QueryContext& context) {
   parseYumConf(kYumConf, results, repos_dir);
 
   std::vector<std::string> sources;
-  if (!resolveFilePattern(repos_dir + "/%.list", sources, GLOB_FILES)) {
-    VLOG(1) << "Cannot resolve yum conf files under " << repos_dir << "/*.list";
-    return results;
-  }
-
-  for (const auto& source : sources) {
-    parseYumConf(source, results, repos_dir);
+  for (const auto& ext : kYumConfigFileExtensions) {
+    if (resolveFilePattern(repos_dir + "/%" + ext, sources, GLOB_FILES)) {
```

Comment:
This works fine, assuming you want to bail out whenever you find either `.list` or `.repo`. However can you please figure out whether the globing supports something like bash's `/*.{list,repo}` so that you could do in one go? Thanks.

Reply:
I looked up in the code base, confirmed that the brace expansion is supported. Here's an example: osquery/filesystem/tests/fileops_tests.cpp:704. Will change the code according your comment.